ncm-cron/src/main/perl/cron.pm

```diff
@@ -453,76 +511,85 @@ sub cleanCrontabs {
             # Not yet implemented
             next;
         } else { # Linux
-            my $fname = "$name.ncm-cron.cron";;
+            my $fname = "$name$CRON_ENTRY_EXTENSION";;
             $fname =~ s{[/\s]}{_}g;
             $cronEntryNames{$fname} = ''; # Value is meaningless
         }
     }
 
-    # Solaris: Collect the crontabs in /var/spool/cron/crontabs then delete
-    # any ncm-cron entries. These are identified as lines between:
-    #     NCM-CRON START:
-    #     NCM-CRON END:
+    my $crondir = $osname eq "solaris" ? $CRONDIR_SOLARIS : $CRONDIR_LINUX;
+
+    # Files only. See remark about LC::Check::absence below
```

Comment:
I think this is confusing. I have to search forward to find another comment (which may well get deleted in the future) to understand it. If I have understood the comment correctly then I think a better way to say it would be:
```
# Files only. Do not modify listdir() call to return of directories unless 
# CAF::Path::cleanup has also been modified to support directories.
```